apax/data/preprocessing.py

```diff
@@ -110,9 +116,11 @@ def get_shrink_wrapped_cell(positions):
     cell_origin = rmin
     cell = np.diag(rmax - rmin)
     for idx in range(3):
-        if np.all(cell[:, idx] < 10e-3):
+        if cell[idx, idx] < 10e-1:
             cell[idx, idx] = 1.0
 
+    cell.flat[:: cell.shape[1] + 1] += 1
+
```

Comment:
what does this line do?

Reply:
same as `cell.diagonal()` idk why I used that.